docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,306 @@
+
+   ![Select the type of project](media/tizen_web8.png)
+
+3. Select the application profile and version from the drop-down list and click Next.
+
+   ![Select the profile and version](media/tizen_web9.png)
+
+4. Select the **Basic UI** template and click **Next**.
+
+  > **Note**
+  >
+  > The list is different depending on the type and profile of the previously selected project.
+
+  ![Select the project](media/tizen_web10.png)
+
+5. Enter the **Project name** and click **Finish** to complete the project creation in workspace.
+  > **Note**
+  >
+  > packageId is an optional field, it is not mandatory to enter details in this field because the packageId will generate the details automatically.
+
+  ![Enter the name of project](media/tizen_web11.png)
+
+6. If you checked all this, the project is created.
+
+   ![Project is created](media/tizen_web12.png)
+
+### Creating Tizen Certificate
+
+To create Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Create Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > If it is created properly, profilename.p12 will be created in tizen-studio/keystore/author.
+
+To add Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Add Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > You can verify the added certificate by running **Certificate Manager**.
+
+To Remove your Tizen certificate:
+
+If you want to change the Certificate, you need to remove the added Certificate.
+
+  1. Open **Command Palette** and enter **Remove Tizen Certificate**.
+  2. Enter the name of the certificate.
+  > **Note**
+  >
+  > When you run Certificate Manager the certificate will be deleted.
+
+### Building Project
+
+To build project:
+
+  1. Open **Command Palette** and enter **Build Tizen App**.
+  2. Enter the folder name of the project you want to build. (Folder name, not path)
+  3. When the build is complete, it succeeds, and there is successful generation of wgt package popup. Also, you will see a build success log in the terminal at the bottom on the vscode.
+
+  ![Building your project](media/tizen_web13.png)
+
+> **Note**  
+> 
+> If you have a Tizen device and you want to deploy the application into it, you must first generate, and then add a certificate.
+
+### Deploying and Running Application in Emulator
+
+To deploy and run application, follow these steps:
+
+1. Open **Command Palette** and enter **Run Emulator-Manager**. The Tizen Emulator Manager is launched.
+
+   > **Note**  
+   > You cannot launch an emulator directly from VS Code. Instead, you must launch the Tizen Emulator Manager and use it to launch the emulator you need.
```

Comment:
Please always use Note style in the following format:
/>Note
/>
/>"text......."

Reply:
VS Code is correct.

done.